nanopub/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .nanopub import Nanopub
+from .models import Nanopub
```

Comment:
I am not completely convinced with the name `models.py`. Any thoughts?

Reply:
Hmm, ```nanopub_class.py```?

Reply:
```nanopublication```? We could also change to class name to ```Nanopublication``` to avoid a similar confusion with the name of the library?

Reply:
Yeah I like that. I've renamed the module to `publication` and the class name to `Publication`. (As it is clear it is a nanopublication because of the nanopub namespace). If you like it please merge!